htdocs/core/modules/product/doc/pdf_standard.modules.php

```diff
@@ -584,6 +589,45 @@ public function write_file($object, $outputlangs, $srctemplatepath, $hidedetails
 		}
 	}
 
+	/**
+	 *  Insert 4 pictures into document.
+	 *
+	 *  @param	TCPDF		$pdf	 		Object PDF
+	 *  @param  object		$object	 	Object to show
+	 *  @param  Translate	$outputlangs	Object lang for output
+	 *  @param	int			$nexY			Position Y to start
+	 *
+	 *  @return	void
+	 */
+	private function _photos(&$pdf, $object, $outputlangs, &$nexY)
+	{
+		global $default_font_size, $conf, $langs;
+		$objphoto = new Product($this->db);
+		$objphoto->fetch($object->ref);
```

Comment:
Do you really need to make the fetch ?
It seems $object is already the $objphoto you need ?